script.js

```diff
@@ -11,3 +11,108 @@ hamburger.addEventListener('click', () => {
+headerLanguages.innerHTML = language;
+
+const workCards = [{
+  cardId: 1,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 2,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 3,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 4,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 5,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+{
+  cardId: 6,
+  cardTitle: 'Proessional Art Printing Data',
+  cardDescription: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry's standard",
+  cardImg: 'image2.png',
+  cardLanguages: ['html', 'bootstrap', 'Ruby'],
+},
+];
+
+const projectCards = document.querySelector('.main-container');
+projectCards.innerHTML = '';
+for (let i = 0; i < workCards.length; i += 1) {
+  let language = '';
+  workCards[i].cardLanguages.forEach((cardLanguages) => {
+    language = `${language}<li>${cardLanguages}<li>`;
+  });
+  projectCards.innerHTML += `<div class="container3">
+  <h1 class="show-text1">${workCards[i].cardTitle}</h1>
+  <p class="paragraph4">${workCards[i].cardDescription}</p>
+  <ul class="lang-list2">${language}</ul>
+  <button class="btn1 display" type="button">See Project</button>
+  </div>
+  `;
+}
+
+const popUpWindow = document.querySelector('.popupcontainer');
+const openPopUp = document.querySelector('.btn1');
+const displayCard = document.querySelector('.display');
+const closeBtn = document.querySelector('.closebtn');
+openPopUp.addEventListener('click', () => {
+  popUpWindow.classList.toggle('active');
+});
+for (let i = 0; i < displayCard.length; i += 1) {
+  displayCard[i].addEventListener('click', () => {
+    popUpWindow.classList.toggle('active');
+  });
+}
+closeBtn.addEventListener('click', () => {
+  popUpWindow.classList.remove('active');
+});
+
+// eslint-disable-next-line no-unused-vars
+function togglePopup() {
```

Comment:
- [ ] Arrow function is the best practiced being used in the industry now, I would implore you to take advantage of such best and industrial standard so it would be part of you.